src/main/java/com/budgetapp/controllers/UserController.java

```
@@ -48,4 +49,14 @@ public ResponseEntity<User> findUserById(@PathVariable final Long id) {
         final User user = userService.findUserById(id);
         return ResponseEntity.ok(user);
     }
+
+    @PostMapping(path = "/login", produces = MediaType.TEXT_PLAIN_VALUE)
+    public ResponseEntity<String> login(@RequestParam final String username, @RequestParam final String password) {
+            final User Authuser = userService.authenticatingUser(username, password);
+        if (Authuser != null) {
+            return ResponseEntity.ok("Successful");
+        } else {
+            return ResponseEntity.notFound().build();
```

General API standards for not being authenticated should return a 401 error not a 404